kaminari-cells.gemspec

```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |spec|
   spec.description   = %q{Simple Kaminari pagination in Cells.}
   spec.homepage      = ""
   spec.license       = "MIT"
+  spec.metadata["rubygems_mfa_required"] = "true"
```

Comment:
Do we really need that?

Reply:
Of course not, it's recommended though. That's why I made a separate commit for this. If you want to we can just drop it.

Reply:
I dropped the 2FA commit.